pyproject.toml

```diff
@@ -5,24 +5,31 @@ description = "Objectherkenning Openbare Ruimte is a project about recognising o
 authors = [
     "Sebastian Davrieux <s.davrieux@amsterdam.nl>",
     "Diana Epureanu <d.epureanu@amsterdam.nl>",
-    "Andrea Lombardo <a.lombardo@amsterdam.nl>"
+    "Andrea Lombardo <a.lombardo@amsterdam.nl>",
+    "Daan Bloembergen <d.bloembergen@amsterdam.nl>"
 ]
 
 [tool.poetry.dependencies]
 python = "~3.8"
 pre-commit = {version = "^3.2.0", source = "PyPI"}
 pydantic = {version = "^1.8.2", source = "PyPI"}
 azure-iot-device = {version = "^2.13.0", source = "PyPI"}
+azure-ai-ml = {version = "^1.15.0", source = "PyPI"}
 azure-storage-blob = {version = "^12.19.0", source = "PyPI"}
 aml_interface = { git = "https://github.com/Computer-Vision-Team-Amsterdam/AML-Interface.git",  tag = "v1.0.0" }
-azure-ai-ml = {version = "^1.2.0", source = "PyPI"}
```

Comment:
Why version 1.2 was removed in favor of 1.15?

Reply:
Because I didn't pay attention. The toml file keeps giving me a headache when merging main into my branch :D Will fix it.